variables.tf

```diff
@@ -61,6 +66,18 @@ variable "image" {
   description = "Docker image to run in the ECS cluster"
 }
 
+variable "load_balancer_internal" {
```

Comment:
I also like load_balander better. But there was already a variable called loadbalancer_eip it seems. The difference is really triggering 😂